pyproject.toml

```diff
@@ -17,8 +17,25 @@ redis = {version = "~=3.*", optional = true}
 redis = ["redis"]
 
 [tool.poetry.dev-dependencies]
+# Testing tools
 fakeredis = "^1.1.0"
 
+# Linting tools
+flake8 = "^3.7.9"
+isort = "^4.3.21"
```

Comment:
I think `isort` released version 5 since this PR was opened and it has some config change. See https://pycqa.github.io/isort/CHANGELOG/. Might be worth it to have a pass to get us straight on there.

Reply:
`isort` version 5 requires Python 3.6+, while this project still supports Python 3.5. I'm therefore going to skip this, however I agree we should revisit this once we're on a more recent Python version.

Reply:
👍  Ah yeah I missed that.